examples/RAmPART_examples/Banana/bananaLLH.m

```diff
@@ -0,0 +1,9 @@
+function llh = bananaLLH( theta )
+
+   t = linspace(0,1,101);
+   X = theta(1) * (1-exp(-theta(2) * t));
+   d = [0.00300970067893022 0.00665959226569197 -0.000955950427737410 0.0163200999871406 0.00630602720690914 -0.0113825158147590 -0.00541481902709891 0.00520038980150608 0.0213718069496413 0.0165729323023027 0.0232420295073312 0.0208637190206789 0.0229973805721590 0.0169323348150834 0.0312731233767299 0.0441180311626104 0.0337876512714904 0.0261245421794418 0.0585111832733452 0.0281102293169161 0.0313127387645332 0.0477757581727388 0.0575986721217150 0.0480583799340222 0.0535880547905150 0.0288771428106876 0.0380008942508774 0.0708768117409631 0.0580506480141715 0.0649177868005986 0.0465307008662812 0.0669660840334922 0.0656588849465678 0.0603452480842966 0.0636771852352871 0.0523249865852781 0.0831089656285460 0.0761292965515737 0.0818340975029384 0.0698866729662230 0.0741613659134849 0.0719491653234356 0.107546088924441 0.0754271026882274 0.109443863087684 0.0849665566507211 0.0913754504323348 0.0865790806861904 0.0780914333054673 0.0969360691902993 0.104142666379560 0.102941802122591 0.104188867997799 0.0869133853987718 0.0850951160753666 0.113564114081587 0.104088928285487 0.105699556714389 0.112702892687693 0.111967675037476 0.112046606310626 0.116397747568882 0.131443915571569 0.127129227885255 0.106850088027157 0.119892136303722 0.117884392018306 0.123676811165650 0.125052463703964 0.139496731291039 0.125390540811436 0.129042907598063 0.145089110257380 0.134118368870722 0.158567776816641 0.131559932635721 0.124511364184778 0.134851088606248 0.160817124740244 0.152132693272153 0.144366600131027 0.158374329532566 0.153739461371015 0.154566591192723 0.179113156957748 0.159129507645844 0.144707318655566 0.156626932077213 0.143620191199112 0.152862655107254 0.174700867169434 0.170788005335972 0.166094156833030 0.177886133214851 0.187948883502206 0.173988206183158 0.190421263016800 0.171501833631737 0.175079742251658 0.177538313202452 0.181050497495496];
+   theta(3) = -1.5229;
```

Comment:
Why is theta(3) overwritten? I recommend using some other constant name instead.